Hublsoft.Net.LoginLogout.DataAccess/RegisteredUsersRepository.cs

```diff
@@ -105,5 +105,65 @@ public async Task IncrementFailedLoginAttemptsAsync(int id)
                 await con.CloseAsync();
             }
         }
+
+        public async Task<int> GetFailedLoginAttempsAsync(int id)
+        {
+            if (id == 0)
+            {
+                throw new Exception("Id cannot be 0");
+            }
+
+            var attempCount = 0;
```

Comment:
typo in variable name, should be `attemptCount`

Reply:
Good spot thanks!